package.json

```diff
@@ -82,16 +83,18 @@
   },
   "scripts": {
     "once": "npm run bundle && npm run firefox",
-    "clean": "rm -rf data/content/*",
+    "setup": "npm-run-all setup:*",
+    "setup:clean": "rm -rf data/content/* && mkdir -p data/content",
+    "setup:externals": "cat node_modules/react/dist/react.min.js node_modules/react-dom/dist/react-dom.min.js node_modules/moment/min/moment.min.js > data/content/vendor.js",
```

Comment:
Doesn't webpack do this stuff magically?
1. https://webpack.github.io/docs/code-splitting.html#split-app-and-vendor-code
2. http://survivejs.com/webpack/splitting-app-and-vendors/